src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
```

Comment:
[CODE KNOWLEDGE] - When creating a method in an object, you don't need to use the 'function' keyword, use the method shorthand instead.

BAD EXAMPLE:
add: function (num) {
  // your code
},

GOOD EXAMPLE:
add(num) {
  // your code
},